subiquity/server/apt.py

```diff
@@ -148,7 +150,15 @@ async def mount(self, device, mountpoint, options=None, type=None):
         return m
 
     async def unmount(self, mountpoint):
-        await self.app.command_runner.run(['umount', mountpoint])
+        for _ in range(5):
+            try:
+                await self.app.command_runner.run(['umount', mountpoint])
+            except subprocess.CalledProcessError as cpe:
+                _cpe = cpe
```

Comment:
We might want to log the unsuccessful attempts (maybe as a warning) so we don't forget to fix the root cause later?